app/components/topic-card/template.hbs

```diff
@@ -4,11 +4,34 @@
   {{/if}}
   {{topic.description}}
 </div>
-<button class='toggle-button-counter' disabled={{session.notAuthenticated}} {{action 'upvote'}}>
-  <div class='icon upvote {{if isUpvoter 'active'}}'></div>
-  <div class='count'>{{topic.upvoteCount}}</div>
-</button>
-<button class='toggle-button-counter' disabled={{session.notAuthenticated}} {{action 'volunteer'}}>
-  <div class='icon volunteer {{if isVolunteer 'active'}}'></div>
-  <div class='count'>{{topic.volunteerCount}}</div>
-</button>
\ No newline at end of file
+
+{{#if topic.talkBy}}
+  <div class="volunteer-container">
+    <img class='avatar' alt="{{topic.talkBy.username}}" title="{{topic.talkBy.username}}" src='{{topic.talkBy.avatar}}' />
+  </div>
+   {{date-format topic.topicDate format="MMMM Do"}}
+   {{#if talkByCurrentUser}}
+     <button class="link-btn" {{action "cancelTalkBy"}}>
+      [{{fa-icon 'times'}}]
+     </button>
+    {{/if}}
+{{else}}
+  <button class='toggle-button-counter' disabled={{session.notAuthenticated}} {{action 'upvote'}}>
+    <div class='icon upvote {{if isUpvoter 'active'}}'></div>
+    <div class='count'>{{topic.upvoteCount}}</div>
+  </button>
+  <button class="icon-btn" alt="Claim topic" title="Claim topic" {{action "chooseTalkDate"}} disabled={{session.notAuthenticated}}>
+    {{fa-icon "plus-circle"}}
+  </button>
+{{/if}}
+
+{{#if showMeetupDates}}
+  <div class="meetup-dates-container">
+    <button class="link-btn" {{action "chooseTalkDate" meetupDates.previousMeetup}}>
+      {{date-format meetupDates.previousMeetup format="MMMM"}}
+    </button> or
+    <button class="link-btn" {{action "chooseTalkDate" meetupDates.nextMeetup}}>
+      {{date-format meetupDates.nextMeetup format="MMMM"}}
+    </button>?
+  </div>
+{{/if}}
```

Comment:
I like the approach here of confirming which date they want to schedule!  I think I would prefer it to say `Today or Next Meetup (M/D/YYYY)` though.

Also, maybe I did this strange but this shouldn't be `meetupDates.previousMeetup` and `meetupDates.nextMeetup`, it should be `moment().startOf('day').toDate()` or `meetupDates.nextMeetup`.  The way I have it coded, if you are between meetup days then previous and next are back-to-back months, but if today is actually a meetup day, then today's meetup is neither the previous meetup or the next meetup.

For example: on Wednesday (8/19) which is a meetup day, the previous meetup will be in July and the next meetup will be in September, today (8/17) the previous meetup is July and the next meetup is August.